blueprints/data-solutions/vertex-mlops/vertex.tf

```diff
@@ -79,54 +86,52 @@ resource "google_notebooks_runtime" "runtime" {
   }
 }
 
-resource "google_notebooks_instance" "playground" {
-  for_each     = { for k, v in var.notebooks : k => v if v.type == "USER_MANAGED" }
-  name         = "${var.prefix}-${each.key}"
-  location     = "${var.region}-b"
-  machine_type = var.notebooks[each.key].machine_type
-  project      = module.project.project_id
-
-  container_image {
-    repository = "gcr.io/deeplearning-platform-release/base-cpu"
-    tag        = "latest"
-  }
-
-  install_gpu_driver = true
-  boot_disk_type     = "PD_SSD"
-  boot_disk_size_gb  = 110
-  disk_encryption    = var.service_encryption_keys.notebooks != null ? "CMEK" : null
-  kms_key            = var.service_encryption_keys.notebooks
-
-  no_public_ip    = var.notebooks[each.key].internal_ip_only
-  no_proxy_access = false
-
-  network = local.vpc
-  subnet  = local.subnet
+resource "google_workbench_instance" "playground" {
+  for_each = { for k, v in var.notebooks : k => v if v.type == "USER_MANAGED" }
+  project  = module.project.project_id
+  name     = "${var.prefix}-${each.key}"
+  location = "${var.region}-b"
 
-  instance_owners = try(tolist(var.notebooks[each.key].owner), null)
-  service_account = module.service-account-notebook.email
-  service_account_scopes = [
-    "https://www.googleapis.com/auth/cloud-platform",
-    "https://www.googleapis.com/auth/userinfo.email",
-  ]
+  gce_setup {
+    machine_type = var.notebooks[each.key].machine_type
+    container_image {
+      repository = "gcr.io/deeplearning-platform-release/workbench-container"
+      tag        = "latest"
+    }
+    boot_disk {
+      disk_size_gb    = 150
+      disk_type       = "PD_SSD"
+      disk_encryption = var.service_encryption_keys.notebooks != null ? "CMEK" : null
+      kms_key         = var.service_encryption_keys.notebooks
+    }
 
+    disable_public_ip = var.notebooks[each.key].internal_ip_only
 
-  metadata = {
-    notebook-disable-nbconvert = "false"
-    notebook-disable-downloads = "false"
-    notebook-disable-terminal  = "false"
-    notebook-disable-root      = "true"
+    network_interfaces {
+      network = local.vpc
+      subnet  = local.subnet
+    }
+    service_accounts {
+      email = module.service-account-notebook.email
+    }
+    metadata = {
```

Comment:
Finding metadata is not easy: no clear link from the terraform documentation or the API documentation. I would add as comment a link to `https://cloud.google.com/vertex-ai/docs/workbench/instances/manage-metadata` for the full list of supported metadata.